utils/actions/detectCodeSmells.ts

```diff
@@ -79,11 +81,61 @@ export default async function detectConsoleLogs({
         console.log(err);
       });
   }
+
   const latestCommitHash = await getLatestCommitHash();
 
   const commentPromises = diffFiles.map(async (file) => {
     const { additions } = getLineDiffs(file.patch ?? "");
 
+    // Leftover comment RegEx
+    // const leftoverCommentRegex = /^\/\*[\s\S]*?\*\//gm;
```

Comment:
This seems very independent, can we split it as to call it in this function?
Other than that LGTM!

Reply:
Lol we needed the code smell itself. Will remove this!